clients/web/src/lib/components/atoms/form/Input/Input.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts" context="module">
+    export type InputSize = "sm" | "md" | "lg";
```

Comment:
Can this be consolidated into the form types?

Reply:
https://github.com/SprocketBot/sprocket/pull/333/commits/82769efee0558dcc49223eb64f156bf0ecd5adb7